layers/src/main/scala/geotrellis/layers/hadoop/package.scala

```diff
@@ -0,0 +1,9 @@
+package geotrellis.layers
```

Comment:
It's a shame that I didn't notice this regression. We don't have this subproject anymore. Fixed here https://github.com/locationtech/geotrellis/pull/3001